types/vuelidate/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "semicolon": false,
+        "strict-export-declare-modifiers": false,
+        "no-single-declare-module": false
```

Comment:
You shouldn't need this since those are module augmentations (i.e. they're declared inside of a module file).

Reply:
Also, for the sake of repo-wide consistency, turning the other rules off would be appreciated (though isn't strictly required).